sgkit/tests/io/plink/test_plink_writer.py

```diff
@@ -52,6 +52,42 @@ def test_write_plink(shared_datadir, tmp_path, plink_in, fam_sep):
     pd.testing.assert_frame_equal(fam_expected, fam_actual)
 
 
+@pytest.mark.parametrize(
+    "plink_in, fam_sep",
+    [
+        (example_dataset_1, "\t"),
+        (example_dataset_2, " "),
+        (example_dataset_3, " "),
+    ],
+)
+def test_zarr_to_plink(shared_datadir, tmp_path, plink_in, fam_sep):
+    # read plink file as a zarr file then write it out again
+    zarr_path = tmp_path / "plink.zarr"
+    plink_to_zarr(path=shared_datadir / plink_in, output=zarr_path, fam_sep=fam_sep)
+    path = tmp_path / "plink_out"
+    path.mkdir(parents=True, exist_ok=False)
+    zarr_to_plink(zarr_path, path=path)
+
+    # check bed files are the same
+    bed_path = path.with_suffix(".bed")
+    with open((shared_datadir / plink_in).with_suffix(".bed"), "rb") as expected, open(
+        bed_path, "rb"
+    ) as actual:
+        assert expected.read() == actual.read()
```

Comment:
There is `filecmp.cmp` in the standard library if you want to save some typing.

Reply:
Thanks for the tip. I changed the test to use it.